KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -52,6 +52,16 @@ export KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=true
 export KUBEVIRT_DEPLOY_GRAFANA=true
 ```
 
+#### start cluster with ksm enabled
+To enable KSM (Kernel Samepage Merging), please also export the following variables before running `make cluster-up`:
+```bash
+# to tune ksm:
+# KUBEVIRT_KSM_SLEEP_BETWEEN_SCANS_MS - This parameter controls
+# how long KSM should sleep in millisecond between scans.
+# KUBEVIRT_KSM_PAGES_TO_SCAN - This parameter controls how many
+# pages KSM should scan in each pass.
```

Comment:
Can you provide an example of how to use these parameters? Expecially for `KUBEVIRT_KSM_PAGES_TO_SCAN`
Thanks

Reply:
All you need to do is to export it with the number of pages you would like it to scan
```
export KUBEVIRT_KSM_PAGES_TO_SCAN=20
```

These Vars matches  sleep_millisecs and pages_to_scan in https://www.kernel.org/doc/Documentation/vm/ksm.txt

Reply:
Thanks, I mean that this example and the documentation link can be added in the code, so it will remain for future usages. Thanks

Reply:
Done :)